components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -214,6 +257,24 @@ export function CreateWorkspacePage() {
                         {isLoading ? "Loading ..." : isStarting ? "Creating Workspace ..." : "New Workspace"}
                     </Button>
                 </div>
+                {existingWorkspaces.length > 0 && (
+                    <div className="w-full flex flex-col justify-end px-6">
+                        <p className="mt-6 text-center text-base">Running workspaces on this revision</p>
```

Comment:
**question**: Definitely out of the scope of this PR, but I wonder if including also stopped workspaces could make sense here. If a user has a stopped workspace on the same branch (revision), does it make sense to also include it here?